etest/tests.py

```diff
@@ -137,13 +137,14 @@ def run(self):
 
             tag_name = str(self.commands.index(command))
 
+            commit_name = self.name.replace("=", "").replace("[", "").replace("]", "")
```

Comment:
We should move this type of sanitization into the commit function itself.

Reply:
That would only work if commit returns the name of the image, as youve said.